src/utilities/searchConfig.js

```diff
@@ -0,0 +1,7 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import lookupConfig from '../../static/lookupConfig.json'
+
+export const findSearchConfig = searchUri => lookupConfig.find(configItem => configItem.uri === searchUri)
```

Comment:
Confusing to call this method findSearchConfig when actually finding in lookupConfig.

Reply:
Do you mind if I rename the whole thing to `authorityConfig`?   It is used for both lookup and searching.

Reply:
That works for me.